mobile-simulation/.gitignore

```diff
@@ -1,5 +1,5 @@
 # Top level directories to ignore
-/Configurations
```

Comment:
This shouldn't be commented out here. That was only to allow you to push the desired config files, we won't need to do that anymore.